src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerTeamOrchestrator.cs

```diff
@@ -214,6 +213,41 @@ public virtual async Task<OrchestratorResponse<AccountDashboardViewModel>> GetAc
                 var tasks = tasksResponse?.Tasks.Where(t => t.ItemsDueCount > 0 && t.Type != "AgreementToSign").ToList() ?? new List<AccountTask>();
                 var showWizard = userResponse.User.ShowWizard && userRoleResponse.UserRole == Role.Owner;
 
+
+                var apprenticeshipResponse = await _mediator.SendAsync(new Queries.GetApprenticeship.GetApprenticeshipRequest
+                {
+                    AccountId = accountResponse.Account.Id
+                });
+
+                int cohortsCount =0, draftApprenticeshipCount =0;
+                CohortSummary singleCohort = new CohortSummary();
+                DraftApprenticeshipDto singleDraftApprenticeship = new DraftApprenticeshipDto();
+                string hashedDraftApprenticeshipId = string.Empty, hashedCohortReference = string.Empty;                

```

Comment:
All of this code (line 217 to 248) would be better in a GetCohortsRequest handler which is called once as per the other mediator calls

e.g.

var cohortsResponseTask = _mediator.SendAsync(new GetAccountCohortsRequest
                {
                    HashedAccountId = hashedAccountId,
                    ExternalUserId = externalUserId
                });

The purpose of the orchestrator is to delegate the work to other services not to do all of this work itself.

Reply:
Done. GetAccountCohortsRequestHandler handles Apprenticeship, Cohorts and DraftApprenticeship.